driver/service.go

```diff
@@ -310,3 +312,16 @@ func (u *upCloudService) waitForStorageState(ctx context.Context, uuid, state st
 		Timeout:      storageStateTimeout,
 	})
 }
+
+func (u *upCloudService) checkIfBackingUp(ctx context.Context, storageUUID string) (bool, error) {
+	storage, err := u.getStorageByUUID(ctx, storageUUID)
+	if err != nil {
+		return false, err
+	}
+
+	if storage.State == upcloud.StorageStateBackuping {
+		return true, nil
+	}
+
+	return false, nil
```

Comment:
Could be less readable, you decide. :)
```suggestion
	return storage.State == upcloud.StorageStateBackuping , nil
```